data_structures/src/superblock.rs

```diff
@@ -192,6 +202,16 @@ impl SuperBlockState {
             hash_prev_block: self.current_superblock_hash?,
         })
     }
+
+    /// Returns the superblock hash and the number of votes of the most voted superblock.
+    /// In case of tie, returns one of the superblocks with the most votes.
```

Comment:
Sounds spooky at first. But then you realize that if we got the same number of votes, it is impossible that they'd be able to meet the 2/3 requirement anyway.

This should be perfectly fine then, unless at some point we act differently depending on whether our locally built superblock matches the one resulting from this function.

Reply:
Yeah that was my though as well. In the future we can add here the logic to check if the most voted superblock has more than 2/3 of the votes, and return `None` otherwise.